packages/config/config/devices/0x008b/trane_xr524.json

```diff
@@ -257,47 +302,57 @@
 		},
 		{
 			"#": "75",
-			"label": "Manually Run Fan",
-			"unit": "minutes",
+			"label": "Recovery Enable",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 120,
-			"defaultValue": 0
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "OFF",
+					"value": 0
+				},
+				{
+					"label": "ON",
+					"value": 1
+				}
+			]
```

Comment:
I'd prefer a label of "Recovery" and options "Enable" / "Disable"

Reply:
OFF/ON is matching XR524 UI and user manual.
Should it match UI/mainual or be enable/disable?

Reply:
We try to use enable/disable everywhere you have an option to toggle. Some UIs even rely on these labels already to display a toggle.